learningresources/tests/test_api.py

```diff
@@ -83,6 +90,11 @@ def test_get_resource(self):
             )
         )
 
+    def test_invalid_create_repo(self):
+        """Create a repository with empty name and description"""
+        with self.assertRaises(IntegrityError):
```

Comment:
Okay, thanks. Should I open an issue, or is "it would be nice" a gentle way of saying it's not important?